app/views/components/_contents-list.html.erb

```diff
@@ -16,20 +16,49 @@
       <%= t("content_item.contents") %>
     </h2>
     <ol class="app-c-contents-list__list">
-      <% contents.each do |contents_item| %>
+      <% contents.each.with_index(1) do |contents_item, position| %>
+      <%
+          active_class = "app-c-contents-list__list-item--active" if contents_item[:active]
```

Comment:
This and the next conditional line would arguably be better expressed in one conditional `if...end` block. My thinking is that it's more obvious if slightly longer.